src/me/StevenLawson/TotalFreedomMod/Commands/Command_trail.java

```diff
@@ -18,16 +18,42 @@
 import org.bukkit.plugin.RegisteredListener;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off]")
+@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off | glass]")
```

Comment:
@JeromSar Maybe change the description to "Pretty glass rainbow trails" or something? I dont know, I think it would be more clear and sound better...